internal/testrunner/runners/system/tester.go

```diff
@@ -842,14 +842,35 @@ type scenarioTest struct {
 	startTestTime      time.Time
 }
 
+type pipelineTrace []string
+
+func (p *pipelineTrace) UnmarshalJSON(d []byte) error {
+	var alias interface{}
+	if err := json.Unmarshal(d, &alias); err != nil {
+		return err
+	}
+	switch v := alias.(type) {
+	case string:
+		*p = append(*p, v)
+	case []any:
+		// asume it is going to be an array of strings
+		for _, value := range v {
+			*p = append(*p, fmt.Sprint(value))
+		}
```

Comment:
Support string and slice of strings.
- logsdb disabled in the stack:
  ```json
  {
    "type": "illegal_argument_exception",
    "message": "unable to convert [eight] to integer",
    "stack_trace": "...",
    "pipeline_trace": [
      "logs-failure_store.test-0.0.1"
    ],
    "pipeline": "logs-failure_store.test-0.0.1",
    "processor_type": "convert"
  }
  ```
- logsdb enabled in the stack:
  ```json
  {
    "message": "unable to convert [eight] to integer",
    "pipeline": "logs-failure_store.test-0.0.1",
    "pipeline_trace": "logs-failure_store.test-0.0.1",
    "processor_type": "convert",
    "stack_trace": "...",
    "type": "illegal_argument_exception"
  }
  ```